hardware/arduino/avr/cores/arduino/Tone.cpp

```diff
@@ -456,19 +457,19 @@ void disableTimer(uint8_t _timer)
 
 #if defined(TIMSK3)
     case 3:
-      TIMSK3 = 0;
+      TIMSK3 &= ~(1 << OCIE3A);
```

Comment:
Actually the problem seems to be TIMSK5... Just tried to compile for the Leonardo:

```
Arduino: 1.6.1 (Linux), Platine: "Arduino Leonardo"

/home/jan/arduino-1.6.1/hardware/arduino/avr/cores/arduino/Tone.cpp: In function 'void disableTimer(uint8_t)':
/home/jan/arduino-1.6.1/hardware/arduino/avr/cores/arduino/Tone.cpp:472:24: error: 'OCIE5A' was not declared in this scope
       TIMSK5 &= ~(1 << OCIE5A);
                        ^
Fehler beim Kompilieren.
```

I'll try to find a fix for that.